releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,7 +6,11 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
-## 1.1.5/1.2.5 - April 27, 2020
+## 1.1.5/1.2.6 - May 28, 2020
+
+* Fixed the bug where configuration refresh didn't work when using provider in Java 11 inside a docker container that is using more than 2 CPUs. Switched from using CompletableFuture to @Async which resolves the issue.
+
+## 1.1.4/1.2.5 - April 27, 2020
```

Comment:
>4 [](start = 7, length = 1)

This was a typo?

Reply:
No, I noticed it was wrong and was updating it.